src/vmm/src/persist.rs

```diff
@@ -161,7 +161,7 @@ pub enum CreateSnapshotError {
 }
 
 /// Snapshot version
-pub const SNAPSHOT_VERSION: Version = Version::new(5, 0, 0);
+pub const SNAPSHOT_VERSION: Version = Version::new(6, 0, 0);
```

Comment:
It feels like we used to call this out in the changelog in the past.

Reply:
Turns out we already bumped the snapshot version since the last release (and that was indeed called out in the changelog). So removed this (but added a commit for adding the 3.0.0 bump from 1.9.0)